config/packages/fos_elastica.yaml

```diff
@@ -124,6 +124,9 @@ fos_elastica:
                                     type: 'text'
                         identifiedStatus:
                             type: 'integer'
+                        coldStorage:
```

Comment:
One of these days, maybe today, we should make properties for these states.
(Also isRestricted, isAvailable, isRemotelyhosted)

Reply:
Also because this exists in submission, couldn't we copy the property instead? Having this property at the dataset level is better maybe?

Reply:
You mean at the entity level? I think we have those already. But we should start using them